### PR TITLE
Add benchmark suite for weiner and inverse functions

### PR DESCRIPTION
**Pull request by abarya**
## Description
The running time for both the functions is approximately 1.5s.

## Checklist
[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
- [x] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- [x] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests

[For detailed information on these and other aspects see [scikit-image contribution guidelines](http://scikit-image.org/docs/dev/contribute.html)]

## References
[If this is a bug-fix or enhancement, it closes issue # ]
[If this is a new feature, it implements the following paper: ]

## For reviewers

(Don't remove the checklist below.)

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

**Comment by abhishekarya286** [comment]
@soupault @jni Please review.

**Comment by soupault** [comment]
Waiting for the decision on https://github.com/scikit-image/scikit-image/pull/3319#issuecomment-410919613.

File: benchmarks/benchmark_filters.py [hunk 1]
```diff
@@ -6,12 +6,22 @@
...
 class FiltersSuite:
     """Benchmark for filter routines in scikit-image."""
+    def _filt_func(self, r, c):
+        return np.exp(-np.hypot(r, c) / 1)
+
     def setup(self):
-        self.image = np.random.random((4000, 4000))
-        self.image[:2000, :2000] += 1
-        self.image[3000:, 3000] += 0.5
+        self.image = np.random.random((600, 600))
```
Comment (emmanuelle) [inline benchmarks/benchmark_filters.py]: why did you decrease the size of the image? I think the idea of having a large enough image was to have an example close to demanding applications, which is more interesting for a benchmark than small images.

**Comment by lagru** [comment]
Hey @abarya, I may be missing some context here but I guess this didn't not get enough attention by reviewers? Sorry for that. Did you close it because you are unable to continue or would you reopen if I were to commit some review time? Otherwise I might make a "Good first issue" out of this. :wink:

**Pull request closed without merging**